webapp/src/properties/updatedTime/__snapshots__/updatedTime.test.tsx.snap

```diff
@@ -5,7 +5,7 @@ exports[`properties/updatedTime should match snapshot 1`] = `
   <div
     class="UpdatedTime octo-propertyvalue octo-propertyvalue--readonly"
   >
-    June 15, 2021, 4:22 PM
```

Comment:
is this change ok? do we want to have it with `at` now?

Reply:
This is because of the `Intl` dependency. But somehow it was failing in `main` branch as well. Not sure why. But as it is a non-blocker change, will look into this in subsequent PR.

Reply:
Sounds good

Reply:
@Rajat-Dabade there was a PR opened to fix webapp tests here a while back https://github.com/mattermost/focalboard/pull/4951 - take a look, maybe it fixes the broken tests

Reply:
@marianunez So after a little deeper investigation I found that this was failing because of bumping up the node's version from 16.10.0 to 21.11 in nvmrc to match it with the latest MM webapp. 

cc: @M-ZubairAhmed